src/sentry/flags/providers.py

```diff
@@ -48,12 +50,87 @@ class InvalidProvider(Exception):
     ...
 
 
+class LaunchDarklyItemSerializer(serializers.Serializer):
```

Comment:
Found the docs for reference - https://apidocs.launchdarkly.com/tag/Audit-log/#operation/getAuditLogEntry
It could be helpful to link this in a comment